src/View/Helper/FormHelper.php

```diff
@@ -43,7 +43,7 @@ class FormHelper extends Helper
         'dateWidget' => '<ul class="list-inline"><li class="year">{{year}}</li><li class="month">{{month}}</li><li class="day">{{day}}</li><li class="hour">{{hour}}</li><li class="minute">{{minute}}</li><li class="second">{{second}}</li><li class="meridian">{{meridian}}</li></ul>',
         'error' => '<div class="invalid-feedback">{{content}}</div>',
         'label' => '<label{{attrs}}>{{text}}{{tooltip}}</label>',
-        'help' => '<small{{attrs}} class="form-text text-muted">{{content}}</small>',
+        'help' => '<small class="form-text text-muted">{{content}}</small>',
```

Comment:
why are the attrs removed here? shouldnt that still be possible?

Reply:
Setting the attrs (e.g. class) will end up in having duplicated class attributes.
E.g. this
`$this->Form->control('email', ['help' => ['class' => 'foo', 'content' => 'someHelp']]);` will end up with
`<small class="foo" content="someHelp" class="form-text text-muted">someHelp</small>`

Since help itself is added as a option to a control the injectClasses() method seems not to work here?!

Reply:
People can set other attributes besides class, for e.g. "id".